Incorrect peer dependency in 1.73.5

### Package

Carbon for IBM Products

### Description

Reported from build error in Jenkins

```
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: @carbon/ibm-products@1.73.5
npm WARN Found: peer carbon-components-react@"^7.70.0" from the root project
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer carbon-components-react@"^7.70.0" from the root project
```

### Component(s) impacted

https://ibm-studios.slack.com/archives/C013ZTX0N6B/p1729520265684109
Can be resolved using resolutions, but should address this for one last clean v1.

### Browser

_No response_

### @carbon/ibm-products (previously @carbon/ibm-cloud-cognitive) version

v1.73.5

### Suggested Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Product/offering

NA

### CodeSandbox or Stackblitz example

https://ibm-studios.slack.com/archives/C013ZTX0N6B/p1729520265684109

### Steps to reproduce the issue (if applicable)

_No response_

### Release date (if applicable)

_No response_

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate issues

--- a/packages/ibm-products/package.json
+++ b/packages/ibm-products/package.json
@@ -97,8 +97,5 @@
     "carbon-icons": "^7.0.7",
     "react": "^16.8.6 || ^17.0.1",
     "react-dom": "^16.8.6 || ^17.0.1"
-  },
-  "resolutions": {
-    "carbon-components-react": "^7.60.0"
   }
 }

--- a/packages/ibm-products/package.json
+++ b/packages/ibm-products/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@carbon/ibm-products",
   "description": "Carbon for IBM Products",
-  "version": "1.73.5",
+  "version": "1.73.2",
   "license": "Apache-2.0",
   "main": "lib/index.js",
   "module": "es/index.js",
@@ -93,7 +93,7 @@
     "@carbon/themes": "^10.55.5",
     "@carbon/type": "^10.45.5",
     "carbon-components": "^10.59.0",
-    "carbon-components-react": "^7.70.0",
+    "carbon-components-react": "^7.60.0",
     "carbon-icons": "^7.0.7",
     "react": "^16.8.6 || ^17.0.1",
     "react-dom": "^16.8.6 || ^17.0.1"

--- a/packages/ibm-products/package.json
+++ b/packages/ibm-products/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@carbon/ibm-products",
   "description": "Carbon for IBM Products",
-  "version": "1.73.2",
+  "version": "1.73.5",
   "license": "Apache-2.0",
   "main": "lib/index.js",
   "module": "es/index.js",
@@ -97,5 +97,8 @@
     "carbon-icons": "^7.0.7",
     "react": "^16.8.6 || ^17.0.1",
     "react-dom": "^16.8.6 || ^17.0.1"
+  },
+  "resolutions": {
+    "carbon-components-react": "^7.60.0"
   }
 }

